Task 16.4.py

```diff
@@ -0,0 +1,11 @@
+class CustomException(Exception):
+    def __init__(self, message):

```

Comment:
`message` is optional in `Exception`, you should account for that. Use *args like in the first task